ArWikiCats/legacy_bots/ma_bots2/mk3.py

```
@@ -0,0 +1,238 @@
+    country: str, arlabel: str, suf: str, year_labe: str, country_label: str, Add_In: bool, arlabel2: str
+) -> tuple[str, bool, bool]:
+    """Handle cases where a year prefix needs a linking preposition."""
+    logger.info("a<<lightblue>>>>>> Add year before")
+
+    to_check_them_tuble = {
+        "Add_in_table": Add_in_table,
+        "add_in_to_country": add_in_to_country,
+        "Films_O_TT": Films_O_TT,
+    }
+
+    co_in_tables, tab_name = check_key_in_tables_return_tuple(country, to_check_them_tuble)
```

Ruff (0.14.13)

Unpacked variable `tab_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)